script/.build-deb/rules

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/make -f
+
+VERSION = $(shell cat VERSION)
+
+override_dh_gencontrol:
+	# if we're on Ubuntu, we need to Recommends: apparmor
+	echo 'apparmor:Recommends=$(shell dpkg-vendor --is Ubuntu && echo apparmor)' >> debian/runc.substvars
+	dh_gencontrol
+
+override_dh_auto_build:
+	make all
+	# ./man/md2man-all.sh runs outside the build container (if at all), since we don't have go-md2man here
+
+override_dh_auto_test:
+	./runc -v
+
+override_dh_strip:
+	# the SHA1 of dockerinit is important: don't strip it
+	# also, Go has lots of problems with stripping, so just don't
```

Comment:
Probably not worth mentioning `dockerinit` here -- just `Go has lots of problems with stripping, so just don't` should be enough. :+1: